elastic/logs/challenges/many-shards-snapshots.json

```diff
@@ -0,0 +1,65 @@
+{% import "rally.helpers" as rally %}
+{
+  "name": "many-shards-snapshots",
+  "description": "Indexes a large amount of shards and then sequentially takes a configurable number of snapshots. Used for benchmarks to help identify regressions related to snapshots with high index counts.",
+  "schedule": [
+    {% include "tasks/many-shards-setup.json" %},
+    {
+      "name": "add-many-shards-ilm",
+      "operation": {
+        "operation-type": "create-ilm",
+        "param-source": "add-track-path",
+        "policies": "ilm/many-shards-quantitative"
+      }
+    },
+    {
+      "name": "initial-indices",
+      "operation": {
+        "operation-type": "raw-bulk",
+        "param-source": "initial-indices-source",
+        "detailed-results": true,
+        "initial_indices_count": {{ p_initial_indices_count }},
+        "name": "auditbeatquantitative",
+        "timeout": "10m"
+      },
+      "clients": {{ p_bulk_indexing_clients }}
+    },
+    {
+      "name": "register-snapshot-repository",
```

Comment:
I noticed that no existing track uses the Rally's `delete-snapshot-repository` operation, any idea why? I guess it would be more robust for local execution?

I'm mostly asking for the challenge I'm working on, I don't think this should block this pull request!

Reply:
That's a good question. Such an operation would make sense when >1 iterations of the same challenge run within the same use case. Even then, I am not sure that one needs to delete the repository; I was under the impression that if you re-register the same repository with the exact same parameters, it'll just verify the repository (we could test that). This is probably what happens when several sequential restorations happen -- in the form of several experiments including ES stop/starts -- in logging query benchmarks.

Reply:
Yeah I think it's only useful if you made a mistake when developing the track and want to start from a clean slate.